PowerOn.html

```diff
@@ -28,14 +28,14 @@
   <div class=hiddenUntilLogo>
     <div id=city></div>
     <div id=progressbar-container>
-      <span id=progressbar-message>Preparing to call</span><br>
+      <span id=progressbar-message>Preparing to connect</span><br>
       <progress id=progressbar max=100 value=5></progress>
-      <button class=submit id=skipDialing onClick=skipDialing()>Skip Dialing</button>
+      <button class=submit id=skipDialing onClick=skipDialing()>Skip Connection</button>
     </div>
   </div>
 
   <div class=logoArea>
-    <img class=hiddenUntilClick src=images\logos\Philips.svg alt="Philips-Magnavox Logo">
+    <img class=hiddenUntilClick src=images\logos\Sony.png alt="Sony Logo">
```

Comment:
This change is pointless - the logo will be randomized (and user-selectable) later on.